contracts/CO30_Matias_FlightOfIcarus.Altis/edit_me/briefing.sqf

```diff
@@ -0,0 +1,97 @@
+#include "..\do_not_edit\script_component.hpp"
+/*
+ * Author: Kresky, Jonpas
+ * Adds briefing based on side.
+ * Call from initPlayerLocal.sqf.
+ * Copy to other side briefings if you want to add briefings to those sides.
+ *
+ * Arguments:
+ * 0: Player <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player] call FUNC(briefing);
+ *
+ * New Line:
+ * <br/>
+ *
+ * Text Colour (Hex colour Code)
+ * <font color='#FF0000'>TEXT HERE</font color>
+ *
+ * Clickable Marker Link:
+ * <marker name = 'MARKER_NAME'>MARKER_TEXT</marker>
+ *
+ * Image:
+ * <img image='FOLDER\IMAGE.jpg' width='200' height='100'/>
+ */
+
+params ["_player"];
+
+switch (side _player) do {
+    // BLUFOR briefing
+    case west: {
+        
+    };
+
+    // OPFOR briefing
+    case east: {
+
+    };
+
+    // RESISTANCE/INDEPENDENT briefing -- THIS IS THE BRIEFING YOU WANT TO EDIT FOR YOUR MISSION!
+    case resistance: {
+        _player createDiaryRecord ["Diary", ["Employer", "
+            <br/>Director boards of several companies.
+        "]];
+
+        _player createDiaryRecord ["Diary", ["Situation", "
+            <br/><font color='#D81717'>ENEMY FORCES:</font color>
+            <br/>Unkown
+            <br/>
+            <br/><font color='#1D69F6'>FRIENDLY FORCES:</font color>
+            <br/>Synixe Contractors, Theseus Inc
+            <br/>
+            <br/><font color='#993399'>CIVILIAN CONSIDERATIONS:</font color>
+            <br/>Aside from the pressence of the VIPs, civilian pressence is expected in towns
+            <br/>
+            <br/><font color='#663300'>TERRAIN CONSIDERATIONS:</font color>
```

Comment:
```suggestion
            <br/><font color='#663300'>TERRAIN CONSIDERATIONS:</font>
```

Reply:
Those colors have always been there, they come with the template